src/lineitemstate-csv-parser.js

```diff
@@ -1,14 +1,20 @@
 import { SphereClient } from 'sphere-node-sdk'
+import { userAgent } from 'sphere-node-utils'
 import _ from 'underscore'
 import highland from 'highland'
 import JSONStream from 'JSONStream'
 import csv from 'csv-parser'
+import loadSelfPkg from 'load-pkg'
 import CONS from './constants'
 
 export default class LineItemStateCsvParser {
   constructor (apiClientConfig, logger, config = {}) {
+    const selfPkg = loadSelfPkg.sync(process.cwd())
     this.client = new SphereClient(
-      Object.assign(apiClientConfig, { user_agent: 'csv-parser-orders' })
+      Object.assign(
+        apiClientConfig,
+        { user_agent: userAgent('csv-parser-orders', selfPkg.version) }
```

Comment:
Isn't the point of putting the user agent in a module is that we don't need to read the version ourselves? Now we include the same package and spread the logic.